cmake/modules/hunter_add_host_subdirectory.cmake

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2016 Fabian Winnen
+# All rights reserved.
+
+# Adds a subdirectory with CMakeList.txt as seperate build with default (host) toolchain
+
+# Usage:
+#   hunter_add_host_subdirectory(host)
+#   hunter_add_host_subdirectory(host
+#       BINARY_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/host-build
+#       ENV_VARS CC CXX
+#   )
+
+# Note:
+#   With polly Android toolchain this fails if you do not unset CC and CXX ENV{} variables.
```

Comment:
What kind of error do you have? To clean environment you can use https://github.com/ruslo/hunter/blob/master/scripts/clear-all.cmake module. Also I personally think that `CC`/`CXX` is completely "autotoolish" stuff and users should use CMAKE_CXX_COMPILER instead. Note that if you want to change compiler for all packages using toolchain is a requirement for Hunter.